src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
+  min-width: 1023px;
+  max-width: 1201px;
+  height: 60px;
```

Comment:
Setting a fixed height for the `html` element can lead to layout issues, especially on smaller screens. Consider using a more flexible layout approach.